versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/spi/AbstractDatabaseAdapter.java

```diff
@@ -635,23 +636,77 @@ protected void validateHashExists(OP_CONTEXT ctx, Hash hash) throws ReferenceNot
    */
   protected abstract List<CommitLogEntry> fetchPageFromCommitLog(OP_CONTEXT ctx, List<Hash> hashes);
 
+  private List<CommitLogEntry> fetchPageFromCommitLog(
```

Comment:
Mind renaming this to `fetchMultipleFromCommitLog`? I think, the term "page" is wrong here (and actually always was - it was just used to fetch a whole page).

Reply:
renamed (in a separate commit for ease of review)